src/core/openthread.cpp

```diff
@@ -66,9 +66,6 @@ static otDEFINE_ALIGNED_VAR(sInstanceRaw, sizeof(otInstance), uint64_t);
 otInstance *sInstance = NULL;
 #endif
 
-void OT_CDECL operator delete(void *, size_t) throw() { }
-void OT_CDECL operator delete(void *) throw() { }
```

Comment:
But they aren't only needed for Windows. As I stated, Linux needs them as well (see `otInstanceFinalize`). If there are duplicate definitions, where are they coming from? Are you using OpenThread in another project that is defining the global new/delete?